Build error on OSX: 'util/ralloc.h' file not found

On OSX 10.10.1 I get the following build error trying to install the current HEAD of master:

```
$ npm install git+https://github.com/aras-p/glsl-optimizer.git
|
> glsl-optimizer@0.1.0 install ./node_modules/glsl-optimizer
> node-gyp rebuild

  CC(target) Release/obj.target/glsl_optimizer_lib/src/glsl/glcpp/glcpp-lex.o
In file included from src/glsl/glcpp/glcpp-lex.l:29:
../src/glsl/glcpp/glcpp.h:32:10: fatal error: 'util/ralloc.h' file not found
#include "util/ralloc.h"
         ^
1 error generated.
make: *** [Release/obj.target/glsl_optimizer_lib/src/glsl/glcpp/glcpp-lex.o] Error 1
```

--- a/src/glsl_optimizer_lib.gyp
+++ b/src/glsl_optimizer_lib.gyp
@@ -7,6 +7,7 @@
       'target_name': 'glsl_optimizer_lib',
       'type': 'static_library',
       'include_dirs': [
+        '.',
         'glsl',
         'mesa',
         '../include',
@@ -138,8 +139,6 @@
         'glsl/opt_flip_matrices.cpp',
         'glsl/opt_dead_builtin_varyings.cpp',
         'glsl/program.h',
-        'glsl/ralloc.c',
-        'glsl/ralloc.h',
         'glsl/s_expression.cpp',
         'glsl/s_expression.h',
         'glsl/standalone_scaffolding.cpp',
@@ -153,8 +152,6 @@
         'mesa/main/dd.h',
         'mesa/main/glheader.h',
         'mesa/main/glminimal.h',
-        'mesa/main/hash_table.c',
-        'mesa/main/hash_table.h',
         'mesa/main/imports.c',
         'mesa/main/imports.h',
         'mesa/main/macros.h',
@@ -167,11 +164,15 @@
         'mesa/program/prog_statevars.h',
         'mesa/program/symbol_table.c',
         'mesa/program/symbol_table.h',
+        'util/hash_table.c',
+        'util/hash_table.h',
+        'util/ralloc.c',
+        'util/ralloc.h',
       ],
       'conditions': [
         ['OS=="win"', {
           'include_dirs': [
-            '../include/c99',          
+            '../include/c99',
           ],
           'defines': [
             '_LIB',
@@ -184,7 +185,7 @@
           ],
           'msvs_disabled_warnings': [4028, 4244, 4267, 4996],
         }],
-      ],        
+      ],
     }
   ]
 }

